realis.go

```diff
@@ -38,6 +38,7 @@ import (
 
 const VERSION = "1.2.1"
 
+// TODO(rdelvalle): Move documentation to interface in order to make godoc look better/more accessible
```

Comment:
Maybe we can say what implementation we have here so user can go check the impl's doc, I suppose we only have one  implementation using thrift protocol though ...

Reply:
Yeah, I think what I want to do is create a testing interface. That way they can use that in their golang tests suites. When we do that then we should definitely add documentation.